framework/src/utils/InputParameters.C

```diff
@@ -277,6 +279,20 @@ InputParameters::isPrivate(const std::string &name) const
 }
 
 void
+InputParameters::declareControllable(const std::string & input_names)
+{
+  std::vector<std::string> names;
+  MooseUtils::tokenize<std::string>(input_names, names, 1, " ");
```

Comment:
I like the use of `MooseUtils` here... it reads more straightforwardly than the complicated `std` way of doing this...